broker/cloud_functions/lite/deployment.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash
+# Deploys or deletes broker Cloud Function
+# This script will not delete Cloud Functions that are in production
+
+testid="${1:-test}"
+# "False" uses production resources
+# any other string will be appended to the names of all resources
+teardown="${2:-False}"
+# "True" tearsdown/deletes resources, else setup
+survey="${3:-ztf}"
+# name of the survey this broker instance will ingest
+versiontag="${4:-v3_3}"
+zone="${CE_ZONE:-us-central1-a}" # use env variable CE_ZONE if it exists
```

Comment:
```suggestion
zone="${5:-us-central1-a}"
```

or delete if unused.